docs/packages/xstate-react/index.md

```diff
@@ -90,6 +90,15 @@ A [React hook](https://reactjs.org/hooks) that interprets the given `machine` an
 
 In the next major version, `useService(service)` will be replaced with `useActor(service)`. Prefer using the `useActor(service)` hook for services instead, since services are also actors.
 
+Also, keep in mind that only a single argument (the event object) can be sent to `send(eventObject)` from `useActor(...)`. When migrating to `useActor(...)`, refactor `send(...)` calls to use only a single event object:
```

Comment:
From the used wording I would think that only event objects are supported but in reality we also support plain strings for "simple" events

Reply:
Works for me!